### ISSUE TITLE
Redis version

### ISSUE THREAD

**Issue by Amplificator** [crazy-max/docker-nextcloud#87]
Hi.

Is it safe to set the Redis docker image to use the latest alpine image which is `redis:7-alpine` or is there a specific reason for the old image being used, perhaps incompability?

I also set mariadb to the latest image supported by Nextcloud which is `mariadb:10.5` and that seems to work fine.

**Comment by crazy-max** [comment]
It depends of the Redis PHP extension which is today in this image the 5.3.6 release which supports Redis 6 and should support Redis 7. See https://github.com/phpredis/phpredis/issues/2068#issuecomment-1025958867. But you're right, I will pin to Redis 6.

**Comment by Amplificator** [comment]
I have done a test with updated versions of all the software needed for a full Nextcloud experience, including (shared) document editing and proper support for Nextcloud Talk using a TURN server.

I have tested and verified the following to not produce any errors:
`traefik:latest`
`mariadb:10.5` (which is the latest version officially supported by Nextcloud)
`redis:alpine` (`alpine` is an image of the latest redis version, currently version 7, running on alpine linux. Verified to work using `docker exec <container id> redis-cli monitor`)
`collabora/code:latest` (server for Nextcloud Office document editing)
`foxcris/docker-coturn:latest` (TURN server for NextCloud Talk)

The only images that should matter is, like you said, redis due to PHP extension support and also mariadb due to a requirement from Nextcloud themselves.
traefik, collabora and coturn is just a matter of setting it up to fit ones needs.
